pkgs/development/libraries/fftw/default.nix

```diff
@@ -5,10 +5,6 @@
 , perl
 , llvmPackages
 , precision ? "double"
-, enableAvx ? stdenv.hostPlatform.avxSupport
```

Comment:
Just out of curiosity: Previously, how did Nix detected if the host platform supports AVX? Did it fail to do so for most users? Does Hydra's CPU considered support AVX?

Reply:
Everything should be defined here: https://github.com/NixOS/nixpkgs/blob/master/lib/systems/architectures.nix

Reply:
If I am not mistaken nothing is enabled by default, for example even if my cpu supports avx: 
```
nix repl --file nixpkgs
nix-repl> stdenv.hostPlatform.avxSupport
false
```

Reply:
FYI: That is set your hostPlatforms arch to one of the values of the architecutures. So as example if you set  `nixpkgs.hostPlatform.gcc.arch = "sandybridge";` if will return `true` for `stdenv.hostPlatform.avxSupport`

Reply:
This setting isn't commonly used, as obviously it would cause you to rebuild everything.

Reply:
Yeah ofc is that rebuilding everything as it will also set `-march=whatever`. I was just trying to explain how those support flags are set.